public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/resourcePickerData.ts

```diff
@@ -69,13 +67,14 @@ export default class ResourcePickerData extends DataSourceWithBackend<AzureMonit
     return resources.map((subscription) => ({
       name: subscription.subscriptionName,
       id: subscription.subscriptionId,
+      uri: `/subscriptions/${subscription.subscriptionId}`, // usually the id is the uri, but for subscriptions it's the opposite
```

Comment:
does this comment still applies? Is the id the uri in any case?

Reply:
lol yeah it's confusing, the id from azure is usually the uri, but this comment is more confusing than helpful so will remove haha